www/js/theme_editor.js

```diff
@@ -63,33 +129,23 @@ const swatchFactory = (themeInstance, isCustom) =>
 
 const swatchHeaderFactory = headerText => h('header.options-header', headerText);
 
-const uploadErrorNotification = (message) => {
-  new Noty({
-    type: 'error',
-    text: message,
-    timeout: 3000,
-    modal: true,
-  }).show();
-};
-
-const imageInput = () =>
+const imageInput = recentBgsContainer =>
   h(
     'label.file-input-label',
     {
       for: 'themebg-upload',
     },
-    'Choose a file',
+    'New Image',
     h('input.file-input#themebg-upload',
       {
         type: 'file',
-        accept: 'image/png, image/jpeg',
         onchange: async (evt) => {
```

Comment:
when selecting the same file twice, nothing happens. thats fine for actual images but when attempting to add a non image file I would expect to see the error each time I try to upload it. is there a different `on-` event that could do that?